ci/prow-entrypoint.sh

```diff
@@ -313,13 +313,13 @@ main() {
             ;;
         "scos-10-build-test-qemu")
             setup_user
-            cosa_init "okd-c10s"
+            cosa_init "c10s"
```

Comment:
Nice. Had an earlier comment about that in the patch series. Should squash this commit into that earlier one.